ports/attr/portfile.cmake

```
@@ -0,0 +1,25 @@
+set(ATTR_VERSION 2.5.1)
```

```suggestion
vcpkg_minimum_required(VERSION 2022-10-12) # for ${VERSION}
```
We can use `${VERSION}` instead of custom `port-version` field. :)